### PR TITLE
fix: picard filtersamreads input

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `<SOFTWARE>.version.txt` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

File: tests/modules/picard/filtersamreads/main.nf [hunk 1]
```diff
@@ -13,15 +13,20 @@ workflow test_picard_filtersamreads {
...
+        .set{ ch_sorted_for_filtersamreads }
+    PICARD_FILTERSAMREADS ( ch_sorted_for_filtersamreads, filter )
 }
 
 workflow test_picard_filtersamreads_readlist {
 
     input = [ [ id:'test', single_end:false ], // meta map
-              file(params.test_data['sarscov2']['illumina']['test_single_end_bam'], checkIfExists: true) ]
+              file(params.test_data['sarscov2']['illumina']['test_single_end_bam'], checkIfExists: true),
+              file(params.test_data['sarscov2']['illumina']['test_single_end_bam_readlist_txt'], checkIfExists: true]
```
Comment (drpatelh) [inline tests/modules/picard/filtersamreads/main.nf:28]:
```suggestion
              file(params.test_data['sarscov2']['illumina']['test_single_end_bam_readlist_txt'], checkIfExists: true) ]
```

Comment (drpatelh) [reply tests/modules/picard/filtersamreads/main.nf:28]: Looks like something is missing here?

**Review by drpatelh — APPROVED** [review]
LGTM! Feel free to merge when the tests pass 👍🏽

**Comment by jfy133** [comment]
Oops, just noticed was not from my fork, my bad :facepalm:

**Pull request merged**